frontend/src/views/setup/Setup.vue

```diff
@@ -118,13 +120,13 @@
 
 <script setup lang="ts">
 
-import {ref} from "vue";
-import axios, {AxiosError} from "axios";
-import {useMainStore} from "@/stores/main";
-import {api} from "@/api";
-import {useRouter} from "vue-router/composables";
+import { ref } from "vue";
+import axios, { AxiosError } from "axios";
+import { useMainStore } from "@/stores/main";
+import { api } from "@/api";
+import { useRouter } from "vue-router/composables";
 import mixpanel from "mixpanel-browser";
-import {TYPE} from "vue-toastification";
+import { TYPE } from "vue-toastification";
```

Comment:
Do we have a ts/js linter for all of these?

Reply:
Not yet. We plan to do that soon - maybe introduce it together with Vue 3